### PR TITLE
Filter

### PR DESCRIPTION
**Pull request by Mehdi-Rh**
# Description

In this PR I set the functionality of filtering companies by their names.

### CODE REVIEW

File: src/components/home/Home.js [hunk 1]
```diff
@@ -1,29 +1,40 @@
+/* eslint-disable no-unused-vars */
 // eslint-disable-next-line no-unused-vars
```
Comment (elmar8287) [inline src/components/home/Home.js]: - [ ]  You have much linter disables in your code. Please, consider removing them - linters should help you to find the bugs on early steps of the development 👍

File: src/components/home/Home.js [hunk 2]
```diff
@@ -1,29 +1,40 @@
...
-import React, { useEffect } from 'react';
+import React, { useState, useEffect } from 'react';
 // eslint-disable-next-line no-unused-vars
 import { useSelector, useDispatch } from 'react-redux';
 // eslint-disable-next-line no-unused-vars
```
Comment (elmar8287) [inline src/components/home/Home.js]: - [ ] Please, consider that when the app is opening, it is not displaying the list of items. It looks like this:
image

File: src/App.js [hunk 3]
```diff
@@ -2,33 +2,11 @@ import { Routes, Route } from 'react-router-dom';
 // import { Container } from 'react-bootstrap';
 import Home from './components/home/Home';
 import Company from './components/item/Company';
-import './App.css';
```
Comment (elmar8287) [inline src/App.js:5]: - [ ] Please, consider making your design similar as it is required with the original one (https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs))

File: src/App.js [hunk 4]
```diff
@@ -2,33 +2,11 @@ import { Routes, Route } from 'react-router-dom';
 // import { Container } from 'react-bootstrap';
```
Comment (elmar8287) [inline src/App.js]: - [ ] Please, consider creating the tests, as required:
-- App has unit tests for several components of the app and some pure functions in Redux code.
-- App has integration tests using React Testing Library

**Review by elmar8287 — CHANGES_REQUESTED** [review]
Hi, @Mehdi-Rh.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- N/A

## Required Changes ♻️

- [ ] Check the comments under the review.
- [ ] Please, consider recording the video presentation, as it is required in this project
- [ ] Please, consider the Gitflow (merge from `dev` to `main` in this milestone) - the PR should be created exactly in this way
- [ ] Please, consider creating the professional Readme file, considering the [MICOVERSE template](https://github.com/microverseinc/readme-template/blob/master/README.md)
- [ ] Please, consider that the README contains an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/). 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
### Please, remember to tag me (@elmar8287) in your question so I can receive the notification 👍 
giphy

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/home/Home.js (hunk 2 above)
Comment (Mehdi-Rh) [reply src/components/home/Home.js]: Hello @elmar8287  Yeah the id you were using has already reached its free request limit, now it's ok updated a new one

**Review by elmar8287 — CHANGES_REQUESTED** [review]
Hi, @Mehdi-Rh.

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.


 ## Highlights

- N/A
 
## Required Changes ♻️

- [ ] Check the comments under the previous review, they are still not fixed, except removing the linter disables
- [ ] Please, consider recording the video presentation, as it is required in this project
 - [ ] Please, consider the Gitflow (merge from dev to main in this milestone) - the PR should be created exactly in this way
- [ ] Please, consider creating the professional Readme file, considering the [MICOVERSE template](https://github.com/microverseinc/readme-template/blob/master/README.md)
- [ ] Please, consider that the README contains an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
### Please, remember to tag me (@elmar8287) in your question so I can receive the notification 👍 
giphy

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Mehdi-Rh** [comment]
> 

Hello @elmar8287 your code review is very accurate however the deadline to submit about 10 minutes, it's impossible to do those changes, I can't repeat the week, I've already repeated one, otherwise I'll have to leave microverse, I'll have no choice!

**Review by AshakaE — APPROVED** [review]
Hi @Mehdi-Rh , 

### Highlights ✨ 

- Pull request 👌 
- Git flow ✅ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Proceed to the next level 👏 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Mehdi-Rh** [comment]
> Hi @Mehdi-Rh ,
> 
> ### Highlights 
> * Pull request ok_hand
> * Git flow white_check_mark
> 
> Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: Congratulations! tada
> 
> Proceed to the next level clap
> 
> Cheers and Happy coding!clapclapclap
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

@

> Hi @Mehdi-Rh ,
> 
> ### Highlights 
> * Pull request ok_hand
> * Git flow white_check_mark
> 
> Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: Congratulations! tada
> 
> Proceed to the next level clap
> 
> Cheers and Happy coding!clapclapclap
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Mehdi-Rh** [comment]
> Hi @Mehdi-Rh ,
> 
> ### Highlights 
> * Pull request ok_hand
> * Git flow white_check_mark
> 
> Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: Congratulations! tada
> 
> Proceed to the next level clap
> 
> Cheers and Happy coding!clapclapclap
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

@AshakaE Thank you very very much for reviewing my project at this early time and for understanding!

**Pull request merged**